ops/docker/ci-builder/check-changed.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Usage: check-changed.sh <diff-pattern>.
+#
+# This script compares the files changed in the <diff-pattern> to the git diff,
+# and writes TRUE or FALSE to stdout if the diff matches/does not match. It is
+# used by CircleCI jobs to determine if they need to run.
+
+set -e
+
+echoerr() { echo "$@" 1>&2; }
+
+# Check if this is a CircleCI PR.
+if [[ -n $CIRCLE_PULL_REQUEST ]]; then
+	PACKAGE=$1
+	# Craft the URL to the GitHub API. The access token is optional for the monorepo since it's an open-source repo.
+	GITHUB_API_URL=$(echo "https://api.github.com/repos/${CIRCLE_PULL_REQUEST:19}?access_token=$GITHUB_ACCESS_TOKEN" | sed "s/\/pull\//\/pulls\//")
```

Comment:
Why is the default value here set to `19`? It should always be set if we make it to this point

Reply:
Fixed